src/sonarr.ts

```diff
@@ -1,136 +1,151 @@
 import { sonarrApi } from './fetch.ts';
 import { cleanTitle, hasExclusionTag } from './utils.ts';
 
+type SonarrApiClient = NonNullable<typeof sonarrApi>;
+
+interface SonarrSeries {
+  id: number;
+  title: string;
+  tvdbId: number;
+  tags: number[];
+}
+
+interface SonarrEpisode {
+  id: number;
+  seriesId: number;
+  tvdbId: number;
+  seasonNumber: number;
+  episodeNumber: number;
+  monitored: boolean;
+}
+
+interface SonarrTag {
+  id: number;
+  label: string;
+}
```

Comment:
These types are already generated from the sonarr API. Also don't think `SonarrTag` is necessary.
https://github.com/Shraymonks/unmonitorr/blob/09469df922552df67329ab25c44539dddfe7a390/src/types/sonarr.d.ts#L9077

https://github.com/Shraymonks/unmonitorr/blob/09469df922552df67329ab25c44539dddfe7a390/src/types/sonarr.d.ts#L8038